src/main/java/com/github/bartimaeusnek/cropspp/crops/cpp/ArditeBerryCrop.java

```diff
@@ -38,16 +38,8 @@ public ItemStack getGain(ICropTile crop) {
 
     @Override
     public int growthDuration(ICropTile crop) {
-        int r;
-        if (ConfigValues.debug)
-            r = 1;
-        else if (crop.getSize() == 2)
-            r = 3000;
-        else if (crop.getSize() == 3 && crop.isBlockBelow("blockArdite"))
-            r = 3000;
-        else
-            r = 500;
-        return r;
+        if (ConfigValues.debug) return 1;
+        return crop.getSize() >= 2 ? 3000 : 500;
```

Comment:
Missing check for size == 3. Intended?

Reply:
Yes, as the code is currently written it always hits on >= 2, so it never hits size == 3 anyway.

Reply:
Could it be a bug that == 3 is not reached  whereas it might be intended to be reachable?

Reply:
It doesn't check for >= 2 here though. Some others do, this and Cobalt not. Though we could decide that we have it the way you changed it to be more in line with the other oreberry crops

Reply:
Could be, that was my first thought; the if/else statements might've just been poorly written.  It would significantly boost the oreberries' growth rate and I didn't want to disrupt any balance (yet), so I just simplifed the branches.

Reply:
> Could it be a bug that == 3 is not reached whereas it might be intended to be reachable?

Probably, though I'm in favor of keeping it as-is too. Partly because nobody complained about OP-ness, partly because it would break existing setups.

Reply:
> It doesn't check for >= 2 here though.

You're right, but it returns 3000 on both 2 and 3 and the block check is handled by canGrow(), so I just combined them.